sunpy/coordinates/transformations.py

```diff
@@ -230,6 +231,15 @@ def _check_observer_defined(frame):
                                "but this is valid for only HeliographicCarrington frames.")
 
 
+def _times_are_equal(time_1, time_2):
```

Comment:
Might be a bit nicer to put this in `sunpy/time/time.py`?

Reply:
I prefer to keep this in `transformations.py`.  It's a helper function with some specialization, with the handling of possible non-`Time` input (namely `None`) and the included use of `np.all()`.  It could call a more general function in `sunpy.time`, but that'll be unnecessary once we have `Time.isclose()` with Astropy 4.2.